### PR TITLE
Add guide for support and on call

### PR DESCRIPTION
**Pull request by evilstreak**
This is a first version of some documentation around our support and on call processes and what it means for the people doing the work.

Huge thanks to @Ellayanor for pulling these words together from various places and getting them into shape 🙏.

Due to the way the headings get turned into a nav sidebar the long question form isn't ideal, but I couldn't bring myself to strip the personality from it to use three word titles. I think it's probably fine:

screenshot 2019-01-16 at 18 19 49

This should be considered a first draft. I expect it to be changed a fair bit in the next month as we get ready to do the first line support rota for real. Given that, I'm mostly interested in hearing about whether there's anything in here you think is wrong or misleading, or shouldn't be made public. Things that are missing or that could be improved should be picked up in another pull request.

I'd particularly like @rjw1, @leedxw and @claredxw to look at this.

### CODE REVIEW

File: guides/support-and-on-call.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+* We provide out-of-hours (24/7) support for some of our clients, responding
+  only to critical alerts and emergency tickets. We call this **on call**, and
+  it’s staffed by a developer or an ops engineer through an opt-in rota.
+
+## Being on first line support
+
+Support tickets come in from clients and users every day. These tickets could
+relate to any of the products we currently support, which are a mixture of
+rails apps and Wordpress implementations, spread across clients all over the
+private sector, including central government.
```
Comment (mec) [inline guides/support-and-on-call.md]: public??

File: guides/support-and-on-call.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+  support**, and it’s staffed by a developer and an ops engineer on rota.
+* We provide out-of-hours (24/7) support for some of our clients, responding
+  only to critical alerts and emergency tickets. We call this **on call**, and
+  it’s staffed by a developer or an ops engineer through an opt-in rota.
+
+## Being on first line support
+
+Support tickets come in from clients and users every day. These tickets could
+relate to any of the products we currently support, which are a mixture of
+rails apps and Wordpress implementations, spread across clients all over the
```
Comment (lozette) [inline guides/support-and-on-call.md]: Capitalise Rails

File: guides/support-and-on-call.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+
+While on support, your focus is on monitoring incoming alerts and tickets,
+responding to them or delegating to colleagues as appropriate.
+
+#### Alerts
+
+We use monitoring and alerts to notify you when an issue occurs, and provide
+[run-books] to help you respond to events and restore service for our clients.
+
+[run-books]: https://git.govpress.com/ops/docs
```
Comment (lozette) [inline guides/support-and-on-call.md]: Should this be an actual link?

File: guides/support-and-on-call.md (hunk 2 above)
Comment (RobjS) [reply guides/support-and-on-call.md]: And "Wordpress" should be "WordPress"

File: guides/support-and-on-call.md (hunk 3 above)
Comment (evilstreak) [reply guides/support-and-on-call.md]: If you mean in terms of markdown syntax, this provides a link for the reference (`[run-books]`) in the paragraph above (and also another paragraph further down the page).

If you mean in terms of that URL doesn't resolve to anything for you, then we'll need to make sure you have the right access permissions for it.

File: guides/support-and-on-call.md (hunk 1 above)
Comment (evilstreak) [reply guides/support-and-on-call.md]: Thanks! Fixed

File: guides/support-and-on-call.md (hunk 2 above)
Comment (evilstreak) [reply guides/support-and-on-call.md]: Thanks both! Fixed

File: guides/support-and-on-call.md (hunk 3 above)
Comment (lozette) [reply guides/support-and-on-call.md]: Ah my mistake, I misunderstood the syntax!

**Comment by evilstreak** [comment]
I've moved all the link references to the bottom. I'll merge when green. Thanks all!

**Pull request merged**